### ISSUE TITLE
Use Twarc as a library to retrieve followers

### ISSUE THREAD

**Issue by alvinyxz** [DocNow/twarc#273]
Hi developers, 

First, thank you for the amazing package that manages the rate limit for us.

I am using Twarc as a library in the notebook to collect an account's followers. After assigning my credentials as object *t*, I used this code:

for ids in t.followers(489348005):
        f= open("489348005.txt","w+")
        for i in ids:
            f.write(i + '\n')

But it returned "AttributeError: 'Twarc' object has no attribute 'followers'" every time. What am I doing wrong? Does the twarc object support followers at all?

**Comment by jeremyboggs** [comment]
@alvinyxz You might try `t.follower_ids` instead of `t.followers`.

**Comment by alvinyxz** [comment]
> @alvinyxz You might try `t.follower_ids` instead of `t.followers`.

Thanks, It works!
For reference:
`with open("nic.txt","w",encoding="utf-8") as f:
        for ids in t.follower_ids(788363167):
            f.write(ids+'\n')`

**Comment by edsu** [comment]
If you do end up wanting the User objects for those user ids you can give the `follower_ids` generator to `lookup_users` like this:

```python
import twarc

twitter = twarc.Twarc()
follower_ids = twitter.follower_ids('barackobama')

for user in twitter.user_lookup(follower_ids):
    print(user['screen_name'])
```

There actually was a bug #274 that was preventing `user_lookup` from working with a generator that was just fixed and released to PyPI as v1.6.4. So you'll need to `pip install --upgrade twarc` for it to work.
